services/getstream/GetstreamService.js

```diff
@@ -16,9 +16,6 @@ class GetstreamService {
     };
     const res = await this._client.feed('topic', id).get(query);
     let data = res.results;
-    if (!data.length) {
-      throw new InvariantError('Topic page not found');
-    }
     return data;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the error check for an empty data array could potentially lead to issues downstream if other parts of the code expect this function to always return a non-empty array. If it's necessary to allow an empty array to be returned, you should ensure that all callers of this function are updated to handle this case correctly.
```diff
- let data = res.results;
+ let data = res.results;
+ if (!data.length) {
+   throw new InvariantError('Topic page not found');
+ }
  return data;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
it is handled in the caller function